src/v/config/node_config.cc

```diff
@@ -169,6 +172,27 @@ node_config::node_config() noexcept
       "injection",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , debug_bundle_write_dir(
+      *this,
+      "debug_bundle_write_dir",
+      "The full path to the directory where debug bundle zip files are stored "
+      "(default: "
+      "/tmp)",
+      {.visibility = visibility::user},
+      "/tmp",
```

Comment:
should we default to somewhere under the data directory?

Reply:
I'm ok with writing to the data dir.  Is the suggestion to keep this config and change the default to the data dir or remove this config option and always write to the data dir?  I'm good with either option.